src/js/bootstrap-slider.js

```diff
@@ -485,7 +486,15 @@ const windowIsDefined = (typeof window === "object");
 						var tick = document.createElement('div');
 						tick.className = 'slider-tick';
 						if (this.options.ticks_tooltip) {
-							this.createTickMouseOverListener(this, tick, i);
+							var tickListenerReference = this._addTickListener();
+                            var enterCallback = tickListenerReference.addMouseEnter(this, tick, i);
+                            var leaveCallback = tickListenerReference.addMouseLeave(this, tick);
+                            if(i === 0){
```

Comment:
So you need to actually map each tick index to an object that contains the function references for its callbacks:

Currently, you are just overwriting the `this.ticksCallbackMap` variable on each iteration of the loop

Try something like the following:
```js
// Generate event listeners for tick
var tickListenerReference = this._addTickListener();
var enterCallback = tickListenerReference.addMouseEnter(this, tick, i);
var leaveCallback = tickListenerReference.addMouseLeave(this, tick);

// Map tick index to event listeners
this.ticksCallbackMap[i] = {
  mouseEnter: enterCallback,
  mouseLeave: leaveCallback
};
```